src/modes/summary/Overview.svelte

```diff
@@ -30,44 +30,21 @@
   $: hospitalTrend = trends[1];
   $: deathTrend = trends[2];
 
-  let minMaxDate = new Date();
+  // Get the latest date that has data for 2 indicators(CASES and HOSPITAL_ADMISSION).
+  // We are not considering DEATHS indicator, because it is weekly based, when other 2 indicators are daily based.
+  // So, we are getting the min date from the max dates of CASES and HOSPITAL_ADMISSION to be sure that we have recent data.
+  $: minMaxDate = new Date(Math.min(...[CASES, HOSPITAL_ADMISSION].map((s) => s.timeFrame.max)));
```

Comment:
Whoops, i didn't realize we weren't considering the `DEATHS` date here...  Can we convert it from a "week" to a "day" so we can compare it?

In the tooltip for "Deaths", it says (as of the time of this comment) "Most recent available date: Sun Apr 14, 2024".  Because "Deaths" is a weekly signal, that date corresponds to the beginning of `EPIWeek` 16 for 2024, and it is valid for the whole week so the data shows up through April 20...

The tooltip seems to get this day value from `maxTime` in https://github.com/cmu-delphi/www-covidcast/blob/084b9be7ba3edec19e8975743b75cef16d73331c/src/stores/DataFetcher.ts#L455, and then i think we can just add 6 days to it??

Reply:
(or if you have a better idea of how to do it, please use that!)

Reply:
Sure, will try to do that!
Thx for mentioning that.

Reply:
@melange396 to add 6 days to it was the easiest solution :)